GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Submodules.cs

```diff
@@ -214,11 +214,6 @@ public SubmoduleTree(TreeNode treeNode, IGitUICommandsSource uiCommands)
 
             protected override void OnAttached()
             {
-                var e = _currentSubmoduleInfo;
-                if (e is not null)
-                {
-                    OnStatusUpdated(e);
-                }
             }
```

Comment:
An empty override can be removed usually. This _is_ the case here since the default implementation is empty.

Reply:
Removed the empty function.
The other overrides seem OK to me, not changing now.